.github/workflows/docs.links.check.config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "ignorePatterns": [
+        {
+            "pattern": "^[^h]"
```

Comment:
If it is necessary to exclude strings starting with `http` or `https`, you can make the regular expression more specific to avoid matching other strings starting with `'h'`, such as `'help'` ...